### ISSUE TITLE
Cannot run codecov with --tries flag

### ISSUE THREAD

**Issue by JungHanter** [codecov/codecov-python#286]
I tried uploading with the below command
```sh
codecov --token=***** --required -X s3 --tries=5
```

The [tries flag] is one of the advanced arguments.

However I got error with this flag.
```sh
codecov: error: unrecognized arguments: --tries=5
```

Is this a bug? Or how can I use this flag?

[tries flag]: https://github.com/codecov/codecov-python/blob/master/codecov/__init__.py#L410

**Comment by nmoinvaz** [comment]
I don't believe there has been a new release. You will have to use the repository. 
```
python -u -m pip install git+https://github.com/codecov/codecov-python.git@master
```

**Comment by JungHanter** [comment]
> I don't believe there has been a new release. You will have to use the repository.
> 
> ```
> python -u -m pip install git+https://github.com/codecov/codecov-python.git@master
> ```

I already use latest version (2.1.7).
I updated again with you command, but the error is still happened.

**Comment by thomasrockhu** [comment]
@JungHanter, the latest version (2.1.7) doesn't have the `tries` argument. I'll release (2.1.8) tomorrow which should have the change.

**Comment by JungHanter** [comment]
> @JungHanter, the latest version (2.1.7) doesn't have the `tries` argument. I'll release (2.1.8) tomorrow which should have the change.

Thanks :)

**Comment by thomasrockhu** [comment]
Hi @JungHanter, we just released `2.1.8`, would you be able to try it?

**Comment by JungHanter** [comment]
Working well. Thanks 😄
